.gitignore

```diff
@@ -21,7 +21,7 @@ testem.log
 /kubernetes/images/frontend/node_Modules/
 .env
 .idea
-
```

Comment:
Don't remove this line.

Reply:
i did not remove it, it was my vscode I guess. i will take care of that.